src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2581,6 +2581,20 @@
   </entry>
 
  <!-- Job Submission stuff -->
+ <entry id="USER_REQUESTED_QUEUE">
+   <type>char</type>
```

Comment:
If you define an empty  <default_value></default_value> you should get None?

Reply:
Line 20 of entry_id.py sets value to "" if value is None - this seems wrong, at the very least it should only do that if the variable is a string.

Reply:
OK, i'll make an issue for this.